NitroxServer/GameLogic/Players/PlayerData.cs

```diff
@@ -182,6 +182,33 @@ public Optional<NitroxId> GetSubRootId(string playerName)
             }
         }
 
+        /// <summary>
+        /// Searches in the persisted playerdata for a NitroxID associated to the given PlayerName
+        /// </summary>
+        /// <param name="playerName">the player to search for</param>
+        /// <returns></returns>
+        public NitroxId GetNitroxId(string playerName)
+        {
+            lock (playersByPlayerName)
+            {
+                PersistedPlayerData playerPersistedData = GetOrCreatePersistedPlayerData(playerName);
+                return playerPersistedData.PlayerNitroxId;
+            }
+        }
+
+        /// <summary>
+        /// Sets a NitroxID corresponding to a given PlayerName in persisted playerdata
+        /// </summary>
+        /// <param name="playerName">the playername to set the ID for</param>
+        /// <param name="id">the NitroxID to set</param>
```

Comment:
While I appreciate comments and documentation, these are only paraphrasing the function name and arguments. Is there anything specific that can be said about the implementation, that is relevant to the caller? Something the documentation _adds on top of_ the method signature?